src/OrchardCore.Modules/OrchardCore.Layers/ViewModels/LayersIndexViewModel.cs

```diff
@@ -14,5 +14,7 @@ public class LayersIndexViewModel
      
         [BindNever]
         public List<Layer> Layers { get; set; }
```

Comment:
Here we have Layers which should have Options which you would define on the Layer.cs class itself?

Reply:
Not sure it does? Are you suggesting adding options to the Layer class? At present the Options are providing a list of available cultures and a selected culture, what need does the Layer class have of that or have I misunderstood?

Reply:
Keep it like that I will analyze further later.